src/detect_breakends.cpp

```diff
@@ -6,34 +6,60 @@ struct cmd_arguments
 {
     std::filesystem::path alignment_file_path{};
     std::filesystem::path insertion_file_path{};
+    std::vector<uint8_t> methods{1, 2, 3, 4};   // default is using all methods
 };
 
 void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments & args)
 {
-    parser.info.author = "David Heller";
+    parser.info.author = "David Heller & Lydia Buntrock";
+    parser.info.app_name = "iGenVar";
+    parser.info.man_page_title = "Short and long Read SV Caller";
     parser.info.short_description = "Detect junctions in a read alignment file";
     parser.info.version = "0.0.1";
+    parser.info.date = "19-01-2021";    // last update
+    parser.info.email = "lydia.buntrock@fu-berlin.de";
+    parser.info.long_copyright = "long_copyright";
+    parser.info.short_copyright = "short_copyright";
+    parser.info.url = "https://github.com/seqan/iGenVar/";
+
+    // Validatiors:
+    // seqan3::value_list_validator method_validator{"1", "cigar_string",
+    //                                               "2", "split_read",
+    //                                               "3", "read_pairs",
+    //                                               "4", "read_depth"};
```

Comment:
I was thinking, if the user wants to insert names instead of numbers, so we could offer both.